cmd/util/fatal.go

```diff
@@ -14,16 +14,17 @@ import (
 var Fatal = fatalError
 
 func fatalError(cmd *cobra.Command, err error, code int) {
-	cmd.PrintErrln()
+	cmd.SetOut(os.Stdout)
+	cmd.Println()
 
 	var bErr bacerrors.Error
 	if errors.As(err, &bErr) {
 		// Print error message
-		cmd.PrintErrln(output.RedStr("Error: ") + bErr.Error())
+		cmd.Println(output.RedStr("Error: ") + bErr.Error())
```

Comment:
why are we switching from stderr to stdout for error messages? It doesn't sound the right thing to do

Reply:
As discussed reverting it.

Reply:
not reverted yet!